docs/_templates/search.html

```diff
@@ -15,173 +15,13 @@
     </div>
 
     <nav id="pagination" class="pagination" aria-label="pagination" hidden="true" >
-        <button class="pagination-previous" id="pagination-previous">
-            ← Previous
-        </button>
-        <ul class="pagination-list"></ul>
-        <button class="pagination-next" id="pagination-next">
-            Next →
-        </button>
+        <div class="pagination-previous" id="pagination-previous"></div>
+        <div class="pagination-item"></div>
```

Comment:
Unless there is a reason otherwise, we should lean towards semantic elements, `ul` and `li` for the inner items make sense here.

Reply:
Added data-attributes to the buttons